src/lib/utils/searchNodes.js

```diff
@@ -0,0 +1,19 @@
+const searchNodes = (nodes, searchText) => {
+  const getNodes = (result, node) => {
+    if (node.label.toLowerCase().includes(searchText.toLowerCase())) {
```

Comment:
It would be better if we move the condition to another variable in order to make things clear.